src/inputdevice.cpp

```diff
@@ -4993,6 +4994,65 @@ static uae_u64 isqual (int evt)
 	return ID_FLAG_QUALIFIER1 << (num * 2);
 }
 
+// Pass the joystick state (joybutton and joydir) to kvbd subsystem and clear them for simulator.
+static void handle_vkbd()
+{
+	if (!vkbd_is_active())
+	{
+		return;
+	}
+
+	for(int joy = 0; joy < MAX_JPORTS; ++joy)
+	{
+		oleft[joy] = 0;
+		oright[joy] = 0;
+		otop[joy] = 0;
+		obot[joy] = 0;
+		horizclear[joy] = 0;
+		vertclear[joy] = 0;
+		mouse_x[joy] = 0;
```

Comment:
@midwan When you say mouse is negatively impacted you mean emulated mouse in Amiberry right? I guess it has to do with these two lines; not sure even why I added them, maybe they are not needed for the vkbd to work.

Reply:
@erikbernsen 
Yes, I mean mouse movement inside the emulation :)
This looks like the most probable cause, as it is moving like the position is reset every time...

Reply:
I removed this lines; it doesn't seem they were needed for the vkbd to work.